src/app/api/egrid/route.ts

```diff
@@ -1,15 +1,69 @@
 import { NextResponse } from "next/server";
 import { fetchEgridData, transformEgridData } from "@/services/egrid-fetch";
+import { addEgridRecord, transformEgridRecord } from "@/services/egrid-store";
 
-/**
- * Route to fetch the egrid data
- * TODO: Implement the route
- * TODO: Add authorization to restrict access to scheduled cron job
- */
 export async function POST() {
```

Comment:
Could we move all of this to the GET route and delete the POST route? We realized that the cron job only works with GET routes and haven't gotten a chance to delete it yet